keystore.go

```diff
@@ -83,3 +84,42 @@ func (km *KeystoreManager) CreateAndSaveKey(keyName string, keyType, bits int) (
 
 	return pk, nil
 }
+
+// ExportKeyToMnemonic is used to take an IPFS key, and return a human-readable friendly version.
+// The idea is to allow users to easily export the keys they create, allowing them to take control of their records (ipns, tns, etc..)
+func (km *KeystoreManager) ExportKeyToMnemonic(keyName string) (string, error) {
+	pk, err := km.GetPrivateKeyByName(keyName)
+	if err != nil {
+		return "", err
+	}
+	pkBytes, err := pk.Bytes()
+	if err != nil {
+		return "", err
+	}
+	phrase, err := mnemonics.ToPhrase(pkBytes, mnemonics.English)
+	if err != nil {
+		return "", err
+	}
+	pkBytes, err = mnemonics.FromPhrase(phrase, mnemonics.English)
```

Comment:
I dont think theres a need to check the phrase is equal to the key - this is something that can be enforced in tests instead